CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+== Contributing Guidelines ==
```

Comment:
This is the WordPress `readme.txt` style... should be `# Contributing Guidelines` ;)

Reply:
Good catch -- thanks! Updated.